core/jazz_ui/build.website

```diff
@@ -0,0 +1,57 @@
+#
+# TODO: Consider moving jq → config
+#
+JQ_BINARY=https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
```

Comment:
`jq` might be available as part of build agent already,. Can you test and download it only if required?